src/original/data.jsx

```
@@ -0,0 +1,49 @@
+          `;
+        document.getElementById("data-container").appendChild(div);
+      });
+    });
+}
+
+function processData(array) {
+  // Inefficient sorting algorithm (O(n^2) complexity for large datasets)
+  for (let i = 0; i < array.length; i++) {
+    for (let j = 0; j < array.length - 1; j++) {
+      if (array[j] > array[j + 1]) {
+        let temp = array[j];
```

Biome

[error] This let declares a variable that is only assigned once.

'temp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)